src/main/java/com/twilio/http/NetworkHttpClient.java

```diff
@@ -86,12 +87,10 @@ public NetworkHttpClient(final RequestConfig requestConfig, final SocketConfig s
     public NetworkHttpClient(HttpClientBuilder clientBuilder) {
         Collection<BasicHeader> headers = Arrays.asList(
                 new BasicHeader("X-Twilio-Client", "java-" + Twilio.VERSION),
-                new BasicHeader(
-                    HttpHeaders.USER_AGENT, "twilio-java/" + Twilio.VERSION + " (" + Twilio.JAVA_VERSION + ") custom"
-                ),
                 new BasicHeader(HttpHeaders.ACCEPT, "application/json"),
                 new BasicHeader(HttpHeaders.ACCEPT_ENCODING, "utf-8")
         );
+        isCustomClient = true;
```

Comment:
What's the purpose of this flag? Would it ever equal to false?

Reply:
it is to know if client(developer) has created custom httpclient. I am using this flag to add " custom" String in UserAgentString.

Reply:
I guess I am missing something here. Won't it be always `true`?

Reply:
By default it will be false, It will be true when NetworkHttpClient instance is created by using constructor:
"NetworkHttpClient(HttpClientBuilder clientBuilder)" (Means by passing custom HttpClientBuilder)

Example:
Check class ProxiedTwilioClientCreator in below mentioned link:
https://www.twilio.com/docs/libraries/java/custom-http-clients-java#

Reply:
Got it. Thanks.